### PR TITLE
added GA tracking events for all the common actions.

### PR DESCRIPTION
**Pull request by mirzu**
You can test with the GA Debug extension. 

Events added for vote, login, reveal, reset

### CODE REVIEW

**Comment by infiniteluke** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://b0de85fd-a329-4d4f-b7b3-ed7c03e5c51a.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/fourkitchens/scrummy-react-dom/branch/ga-simple/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

File: src/index.html [hunk 1]
```diff
@@ -11,5 +11,13 @@
...
       <noscript>You cannot play Scrummy without JavaScript.</noscript>
     </main>
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m    )
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+      ga('create', 'UA-559851-15', 'auto');
+      ga('send', 'pageview');
+</script>
```
Comment (flipactual) [inline src/index.html]: Can we fix the indentation here?

File: test/actions/ScrummyAPI.js [hunk 2]
```diff
@@ -5,6 +5,10 @@ import sinon from 'sinon';
 import thunkMiddleware from 'redux-thunk';
 const mockStore = configureStore([]);
 
+test.beforeEach(() => {
+  global.ga = () => {};
```
Comment (infiniteluke) [inline test/actions/ScrummyAPI.js:9]: Let's move this to https://github.com/fourkitchens/scrummy-react-dom/blob/master/test/helpers/dom.js#L13. That way you don't have to do it twice. You'll be able to delete the `beforeEach()`

File: test/actions/ScrummyAPI.js [hunk 3]
```diff
@@ -5,6 +5,10 @@ import sinon from 'sinon';
 import thunkMiddleware from 'redux-thunk';
 const mockStore = configureStore([]);
 
+test.beforeEach(() => {
```
Comment (flipactual) [inline test/actions/ScrummyAPI.js]: Can just use [`before`](https://github.com/avajs/ava#before--after-hooks) here, right?

File: test/actions/actions.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ import messageTypes from '../../src/actions/messageTypes';
 test.beforeEach(() => {
   window.scrummyAPI = {};
   window.scrummyAPI.emit = sinon.spy();
+  global.ga = () => {};
```
Comment (infiniteluke) [inline test/actions/actions.js]: This can be deleted once ga is stubbed in dom.js

Comment (flipactual) [inline test/actions/actions.js]: This can also be moved to its own `before` hook

**Review by flipactual — CHANGES_REQUESTED** [review]

**Review by infiniteluke — CHANGES_REQUESTED** [review]
Manual test with ga debugger showed events being sent. 🎉 Let's just move those stubs into a central location.

**Comment by infiniteluke** [comment]
This addresses #20. 🎉

**Review by infiniteluke — COMMENTED** [review]
```
  _       _             
 | | __ _| |_ _ __ ___  
 | |/ _` | __| '_ ` _ \ 
 | | (_| | |_| | | | | |
 |_|\__, |\__|_| |_| |_|
    |___/               
```

**Pull request merged**